packages/react/accordion/src/Accordion.tsx

```diff
@@ -59,28 +58,6 @@ const Accordion = React.forwardRef<AccordionElement, AccordionSingleProps | Acco
 
 Accordion.displayName = ACCORDION_NAME;
 
-Accordion.propTypes = {
-  type(props) {
-    const value = props.value || props.defaultValue;
-    if (props.type && !['single', 'multiple'].includes(props.type)) {
-      return new Error(
-        'Invalid prop `type` supplied to `Accordion`. Expected one of `single | multiple`.'
-      );
-    }
-    if (props.type === 'multiple' && typeof value === 'string') {
-      return new Error(
-        'Invalid prop `type` supplied to `Accordion`. Expected `single` when `defaultValue` or `value` is type `string`.'
-      );
-    }
-    if (props.type === 'single' && Array.isArray(value)) {
-      return new Error(
-        'Invalid prop `type` supplied to `Accordion`. Expected `multiple` when `defaultValue` or `value` is type `string[]`.'
-      );
-    }
-    return null;
-  },
-};
-
```

Comment:
Prop types have been deprecated for a while and are now completely removed
https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-deprecated-react-apis

Using TypeScript already handles this